Trainer_v5/Trainer_v5.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Trainer_v5</RootNamespace>
     <AssemblyName>Trainer_v5</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```

Comment:
@muddxyii Can you explain why this upgrade is needed?

Reply:
My mistake, I had it set up wrong on my system and my IDE auto upgraded it when I tried to compile the DLL. I reverted the change.